services/sse-api/Makefile

```diff
@@ -0,0 +1,20 @@
+# environment variables for the commands (docker compose, poetry)
+export COMPOSE_PROJECT_NAME := sse-api
+export MONGO_PORT := 27035
+export CACHE_MONGO_URL := mongodb://localhost:${MONGO_PORT}
+export QUEUE_MONGO_URL := mongodb://localhost:${MONGO_PORT}
```

Comment:
Same here, do we use this config for sse? I mean, it only connects to the cache collection right?